cmd/oceantv/broadcast_hardware_machine.go

```diff
@@ -166,7 +250,37 @@ func (sm *hardwareStateMachine) handleTimeEvent(t timeEvent) {
 		eventIfStatus(hardwareStoppedEvent{}, false)
 	case *hardwareRestarting:
 		eventIfStatus(hardwareStartRequestEvent{}, false)
+	case *hardwareRecoveringVoltage:
+		withTimeout := sm.currentState.(stateWithTimeout)
+		if withTimeout.timedOut(t.Time) {
+			sm.ctx.logAndNotify(broadcastHardware, "voltage recovery timed out")
+			sm.ctx.bus.publish(hardwareStartFailedEvent{})
+			sm.transition(newHardwareOff())
+			return
+		}
 
+		voltage, err := sm.ctx.camera.voltage(sm.ctx)
+		if err != nil {
+			msg := fmt.Sprintf("could not get hardware voltage: %v", err)
+			sm.log(msg)
+			sm.ctx.bus.publish(invalidConfigurationEvent{msg})
+			return
+		}
+
+		// If RequiredStreamingVoltage is not set, default to 24.5.
+		if sm.ctx.cfg.RequiredStreamingVoltage == 0 {
```

Comment:
Ditto. where is RequiredStreamingVoltage set?

Reply:
Also will be available under the advanced broadcast settings.